Java build fails in jni binding folder

Commit : https://github.com/zeromq/czmq/commit/e91c2b01cc15eead5f97deef13ee5d52e3a5a22e 

Issue : java build fails in jni binding folder
------------------------------------------------------------------------
build log
...git/tmp/czmq/bindings/jni$ gradle build


:compileJava UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:generateJniHeaders
:initCMake
-- Configuring done
-- Generating done
-- Build files have been written to: /home/vnd2kor/bezirk/git/tmp/czmq/bindings/jni
:buildNative
[100%] Built target czmqjni
:jar UP-TO-DATE
:assemble UP-TO-DATE
:compileTestJava UP-TO-DATE
:processTestResources UP-TO-DATE
:testClasses UP-TO-DATE
:test
 * ztrie: OK
java: src/zstr.c:336: zstr_test: Assertion `server' failed.
 * zuuid: OK
 * zconfig: OK
 * ziflist: OK
 * zhashx: OK
 * zstr: :test FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> Process 'Gradle Test Executor 1' finished with non-zero exit value 134

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 6.591 secs

--- a/src/zstr.c
+++ b/src/zstr.c
@@ -330,6 +330,7 @@ zstr_test (bool verbose)
     zsock_destroy (&input);
     zsock_destroy (&output);
 
+#if defined (ZMQ_SERVER)
     //  Test SERVER/CLIENT over zstr
     zsock_t *server = zsock_new_server ("inproc://zstr-test-routing");
     zsock_t *client = zsock_new_client ("inproc://zstr-test-routing");;
@@ -379,6 +380,7 @@ zstr_test (bool verbose)
 
     zsock_destroy (&client);
     zsock_destroy (&server);
+#endif
     //  @end
 
     printf ("OK\n");

